frontend/test/metabase/scenarios/admin/settings/email.cy.spec.js

```diff
@@ -32,12 +38,15 @@ describe("scenarios > admin > settings > email settings", () => {
     cy.findByDisplayValue("25");
     cy.findAllByDisplayValue("admin");
     cy.findByDisplayValue("mailer@metabase.test");
+    cy.findByDisplayValue("Sender Name");
+    cy.findByDisplayValue("reply-to@metabase.test, reply-to-too@metabase.test");
```

Comment:
@alxnddr FYI I extended this test but it is currently failing. The current behaviour is such that the input field cannot contain any whitespace.